core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -1928,25 +1927,29 @@ class TaskSetManagerSuite
     // get the TaskSetManager
     val manager = sched.invokePrivate(taskSetManagers()).get(stageId).get(stageAttemptId)
 
-    val task0 = manager.resourceOffer(exec0, "localhost", TaskLocality.NO_PREF)
-    val task1 = manager.resourceOffer(exec1, "localhost", TaskLocality.NO_PREF)
-    assert(task0.isDefined && task1.isDefined)
-    val (taskId0, index0) = (task0.get.taskId, task0.get.index)
-    val (taskId1, index1) = (task1.get.taskId, task1.get.index)
+    val (task0, task1) = eventually(timeout(10.seconds), interval(100.milliseconds)) {
+      val task0 = Option(manager.taskInfos(0))
+      val task1 = Option(manager.taskInfos(1))
+      assert(task0.isDefined && task1.isDefined)
+      (task0, task1)
+    }
+
+    val (taskId0, index0, task0exec) = (task0.get.taskId, task0.get.index, task0.get.executorId)
```

Comment:
Can we rename `task0exec` -> `exec0` and `task1exec` -> `exec1` to reduce the diff?

Reply:
Done